person.rb

```diff
@@ -0,0 +1,28 @@
+require './nameable'
+
+class Person
+  def initalize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] The Person class needs to inherit from the Nameable class.

- [ ] You made a typo error in your spelling. It should be `initialize`, not initalize. Notice the "i" after "t".

  ```
  class Person < Nameable
    def initialize(age, name = 'Unknown', parent_permission: true)
    .
    .
  ```

Reply:
@julie-ify , thanks for the review and corrections... O will correct that immediately